xbmc/GUIInfoManager.cpp

```diff
@@ -2406,6 +2406,15 @@ bool CGUIInfoManager::EvaluateBool(const std::string &expression, int contextWin
   return result;
 }
 
+bool CGUIInfoManager::EvaluateBool(const std::string &expression, const CGUIListItemPtr &item)
```

Comment:
This is pretty much adding duped code. Mind refactoring it so it take context window and/or the listitem?